src/components/arrows/arrows.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './arrows.css';
+import {LEFT, RIGHT} from '../../consts';
+
+const Arrows: React.FC = () => {
+    return (
+        <div className='arrows'>
+            <button className='arrow arrow_left' type='button' >{LEFT}</button>
```

Comment:
для jsx элементов с больше чем 1 пропсом нужно оформлять в несколько строк
```suggestion
            <button 
                className='arrow arrow_left' 
                type='button'
            >
                {LEFT}
            </button>
```